serde/src/java/org/apache/hadoop/hive/serde2/AbstractSerDe.java

```diff
@@ -189,7 +192,7 @@ public Optional<Configuration> getConfiguration() {
 
   @Override
   public String toString() {
-    return "AbstractSerDe [log=" + log + ", configuration=" + configuration + ", properties=" + properties
+    return "AbstractSerDe [log=" + getClassLogger() + ", configuration=" + configuration + ", properties=" + properties
```

Comment:
Hi, just trying to understand, why do we need a getter for a static final variable? Isn't using the object `log` simpler, and then you don't need to override the method `getClassLogger()` everywhere?

Reply:
Let's drop `log` completely from here. This is purely for debugging purposes and I don't think it helps much to see the logger especially now that we are turning everything to static fields.

Reply:
I was making sure that there is no change in the debug message even after changing the logger fields to static final. But yes it makes sense to totally remove this.